cmd/registry/registryList.go

```diff
@@ -100,20 +109,30 @@ func parseRegistryListCmd(cmd *cobra.Command, args []string) ([]*k3d.Node, bool)
 			Name: name,
 		})
 	}
-
-	return nodes, headersOff
+	return nodes
 }
 
-func printNodes(nodes []*k3d.Node, headersOff bool) {
+func printNodes(nodes []*k3d.Node, flags registryListFlags) {
+	// the output details printed when we dump JSON/YAML
+	type jsonOutput struct {
+		k3d.Node
+		Port int `yaml:"port" json:"port"`
```

Comment:
We'll have a duplicate notion of ports here then, right?
Do you think that could be confusing?

Reply:
I must admit that the original intention was to have an `Address` instead of a `Port`, something that could point users to a useful address like `registry.localhost:46223`. But as this proved to be a bit more complicated than I expected (ie, for remote `DOCKER_HOST` we should parse it and extract the host and so on...), I took the shortcut and kept only the Port. I agree that this could be confusing. I think I will remove it...